<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="MainTabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1281</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Energy_Tab">
     <attribute name="title">
      <string>Energy</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>109</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ef_mas_well</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="qw_right_height">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>200</y>
        <width>125</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="remove_sel_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>400</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove QW</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>179</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>eng_step</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="qw_left_height">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>200</y>
        <width>125</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="get_energy_Button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>460</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Energy and Wave Functions</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="eng_step_LineEdit">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>200</y>
        <width>125</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Input_Data</string>
      </property>
     </widget>
     <widget class="qw_input_TextEdit" name="input_data_TextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>256</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>109</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ef_mas_bar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>109</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ef_mas_bar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>179</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>QW_right_height</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>10</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Energies and Wave Functions</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>250</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Well_width</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_well_Button">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>400</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add QW</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>179</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>QW_left_height</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ef_mas_well">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>130</y>
        <width>125</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Barrier_width</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ef_mas_rbar">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>130</y>
        <width>125</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="bar_width">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>270</y>
        <width>125</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="qw_input_TextEdit" name="res_en_wf_PlainEdit">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>31</y>
        <width>256</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ef_mas_lbar">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>130</y>
        <width>125</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="well_width">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>270</y>
        <width>125</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>400</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit QW</string>
      </property>
     </widget>
     <widget class="QPushButton" name="get_transitionsButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>460</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Transitions</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="txt_ui_checkBox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>530</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>txt_ui</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>521</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Рассчет уровней энергии и волновых функций &lt;br/&gt;системы связанных квантовых ям.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>B Field</string>
     </attribute>
     <widget class="QPushButton" name="monteCarlo_Button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Do MonteCarlo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Вычисление 4D мерного интеграла на CUDA.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="M_Field_Tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>B_Field</string>
     </attribute>
     <widget class="QLineEdit" name="B_MIN_Edit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>B_MIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="B_MAX_Edit">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>B_MAX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>350</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Theta_from</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Theta_from">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>380</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>350</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Theta_to</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Theta_to">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>45</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>B_current</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="B_current_Edit">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>40</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="get_Ws_for_B_Button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>610</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Evaluate W's for Current B_Field</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>From Landau Level</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>130</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>nu_i</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nu_i_Edit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>n_i</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="n_i_Edit">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>160</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>130</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>nu_j</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nu_j_Edit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>160</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>120</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>n_j</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="n_j_Edit">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>250</y>
        <width>57</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>nu_g</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>260</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>nu_f</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nu_f_Edit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>290</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nu_g_Edit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>290</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>350</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Step</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Theta_step_Edit">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>380</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Energies</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="energies_for_fields_Edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>311</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Transitions</string>
      </property>
     </widget>
     <widget class="qw_input_TextEdit" name="Ws_for_B_field_Edit">
      <property name="geometry">
       <rect>
        <x>733</x>
        <y>40</y>
        <width>351</width>
        <height>611</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>230</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To Landau Sublevel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>W's for Current B_field</string>
      </property>
     </widget>
     <widget class="QPushButton" name="get_transitions_Button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>550</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Transitions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_field_Button">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>550</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Field</string>
      </property>
     </widget>
     <widget class="qw_input_TextEdit" name="B_fields_Edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>311</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>460</y>
        <width>341</width>
        <height>70</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Переходы получаем такие, что поля в них&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;не выходят за пределы B_MIN, в пределах&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;заданных углов.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Some</string>
     </attribute>
     <widget class="QPushButton" name="plot_averages_Button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot averages.</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>221</height>
       </rect>
      </property>
      <property name="plainText">
       <string>eng_levels: 0,1
left qw: 25
barrier: 2
right qw: 11
eff_mas: 0.067
height: 240.
left well width is vaied: 250 - 50

</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plot_b_angle_Button">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot B_Angle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MyPlotButton">
      <property name="geometry">
       <rect>
        <x>919</x>
        <y>620</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>MyPlot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OpenGL_Button">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>OpenGL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cuda_Button">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>170</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Cuda_OpenGL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OpenGLnew_Button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Cuda_OpenGLnew</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>335</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cuda OpenGL Interoperability.</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="clear_all_Button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>720</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear ALL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>730</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use CUDA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plotButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>720</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PLOT WF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qw_input_TextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">qw_input_textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>936</x>
     <y>787</y>
    </hint>
    <hint type="destinationlabel">
     <x>1001</x>
     <y>757</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
